main.go

```diff
@@ -102,6 +110,43 @@ func handleUserRequest(responseWriter http.ResponseWriter, request *http.Request
 	}
 }
 
+func handleSessionRequest(
+	responseWriter http.ResponseWriter,
+	request *http.Request) {
+
+	type LoginForm struct {
+		EmailAddress string `json:"emailAddress"`
+		Password     string `json:"password"`
+	}
+
+	switch request.Method {
+	case http.MethodPost:
+		var loginForm LoginForm
+		body := getRequestBody(request)
+
+		if err := json.Unmarshal(body, &loginForm); err != nil {
+			panic(err)
+		}
+
+		validated, err := databaseutil.ValidateUser(
+			loginForm.EmailAddress,
+			loginForm.Password)
+
+		if err != nil {
+			panic(err)
+		}
+
+		log.Printf("did we find the user + password combo in the table: %t", validated)
+		responseWriter.WriteHeader(http.StatusCreated)
+		responseWriter.Write([]byte(fmt.Sprintf("passward email combo was correct? %t", validated)))
```

Comment:
is this string getting sent back in the response body?

Reply:
bump

Reply:
yes its being sent back in the response body. I wanted to send back something that contained if the login correct/incorrect. Mostly to test out the flow. I imagine as we progress something different will be sent back